inc/Integrations/Shopify/ShopifyIntegration.php

```diff
@@ -17,6 +17,11 @@ public static function register_blocks(): void {
 
 		foreach ( $data_source_configs as $config ) {
 			$data_source = ShopifyDataSource::from_array( $config );
+
+			if ( false === $config['service_config']['enable_blocks'] ) {
+				continue;
+			}
+				
 			self::register_blocks_for_shopify_data_source( $data_source );
```

Comment:
```suggestion

			if ( false === $config['service_config']['enable_blocks'] ) {
				self::register_blocks_for_shopify_data_source( $data_source );
			}
```

Reply:
Max's code suggestion here is the wrong way 'round